### ISSUE TITLE
TreeMap legend.valueLabels does't work correctly

### ISSUE THREAD

**Issue by Identity-labs** [amcharts/amcharts4#1151]
Related to : https://github.com/amcharts/amcharts4/issues/1084

If you set useDefaultMarker at true on the legend on the first load you don't have the valueLabels displayed, you have to zoom & un zoom to refresh it

**Comment by Identity-labs** [comment]
The fix you have done, I have done too before you,
But I have the bug of the first loading with default marker

```ts
 	public get value(): number {
-		return this.treeMapDataItem.value;
+		let treeMapDataItem = this.treeMapDataItem;
+		if(treeMapDataItem){
+			return treeMapDataItem.value;
+		}
 	}
```

**Comment by martynasma** [comment]
Thanks for the heads up. We'll try to get the fix into next release. (prob at some point early next week)

**Comment by martynasma** [comment]
Fixed in [4.3.7](https://github.com/amcharts/amcharts4/releases/tag/4.3.7).

## [4.3.7] - 2019-04-04

### Fixed
- `TreeMap` was erroring out since `4.3.6` unless "animated" theme was enabled.
- On `TreeMap`'s legend with default markers `valueLabels` was not working on first load.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
